docs/getting-started/setupCFG.md

```diff
@@ -0,0 +1,178 @@
+---
```

Comment:
Can we rename the file to `setup-cfg.md`?

Reply:
Yep. I updated